weather_dl/download_pipeline/clients.py

```diff
@@ -53,6 +53,11 @@ def num_requests_per_key(self, dataset: str) -> int:
         """Specifies the number of workers to be used per api key for the dataset."""
         pass
 
+    @property
+    @abc.abstractmethod
+    def license_url(self):
+        """Specifies the License URL."""
+
```

Comment:
I'm surprised this runs! Doesn't it need a `pass` statement at the very least?

Reply:
The code was running because we had written docstring. We have added pass in said method for better readability.

Reply:
Thanks!